raui-core/src/data_binding.rs

```diff
@@ -86,7 +187,10 @@ where
         }
     }
 
+    /// Set the inner data directly and notify the [`ChangeNotifier`] ( if set )
     pub fn write(&mut self, v: T) {
+        // TODO: This will silently swallow any failure to obtain the RwLock, should we return a
```

Comment:
we can discuss implementing result for the next version later, but for this version we can go with silent failure to not block the update - there will be other `write` methods in other types that might need to return Result too so we can make a bulk change for all writes in RAUI.

Reply:
OK, cool, that's what I was thinking. :+1: